src/pages/getting-started/index.mdx

```diff
@@ -43,8 +43,8 @@ One of the paths is to extend user interfaces. We provide developers with tools
 ## Agent App
 
 <img
-  alt="LiveChat Tag Master"
```

Comment:
How about you wrap` <img>` in `<div style={{marginBottom: 10px}}></div>`? So that there's some space between the image border and the text below it.

Reply:
smart! will do